client/state/analytics/reducer.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import {
+	ANALYTICS_CONTINOUS_MONITOR_ON,
+	ANALYTICS_CONTINOUS_MONITOR_OFF
+} from 'state/action-types';
+
+export const isTracking = ( state = false, { type } ) => {
```

Comment:
> in any case, I think `isSomething` is the de-facto standard in our other reducers, no?

Is it? TBH I've always strived for nouns, so for the deprecated-yet-ubiquitous `isRequestingSomething` selector, my corresponding reducer would be called `somethingRequests` (plural 'cause they're per-site).

Like I said, I thought predicates like `isDoingSomething` were fine for selectors (which are read-only, so to speak), and I thought a different nomenclature for reducers was good to differentiate them easily.